telebot.py

```diff
@@ -0,0 +1,19 @@
+import requests
+import constants
+import urllib
+import io
+
+
+def tbotxt(text):
```

Comment:
Name of this function and function `tbotimg` is kinda tautology. Maybe it's better to rename them to `send_text` and `send_picture`?